hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/schemas/V0490FileSchema.java

```diff
@@ -274,7 +274,7 @@ public void createGenesisFeeSchedule(@NonNull final SystemContext systemContext)
      */
     public Bytes genesisFeeSchedules(@NonNull final Configuration config) {
         final var resourceName = config.getConfigData(BootstrapConfig.class).feeSchedulesJsonResource();
-        try (final var in = Thread.currentThread().getContextClassLoader().getResourceAsStream(resourceName)) {
+        try (final var in = V0490FileSchema.class.getResourceAsStream(resourceName)) {
```

Comment:
In the modules system, _resources_ need to follow the same restrictions as Java classes. If the resources are local to a module, they need to be loaded from inside the Module, which we can do via `Class.getResourceAsStream` where the referenced class is in the same Module as the resource to load. 

`Thread.currentThread().getContextClassLoader()` on the other hand no longer sees the resources as it operates outside of the context of the module.